main_app/templates/main_app/event_list.html

```diff
@@ -10,7 +10,7 @@ <h1>Cosmic Events</h1>
       <ul id="filter-dropdown" class="dropdown-content">
         <li><a href="?orderby=title">By Name</a></li>
         <li><a href="?orderby=start_date">By Date</a></li>
-        <li><a href="?orderby=event_type">By Event Type</a></li>
+        <li><a href="?">By Event Type</a></li>
```

Comment:
@mayaellington It looks like the orderby=event_type was removed in this commit. Can you put it back and update the PR? Thanks!